.gitignore

```diff
@@ -18,6 +18,11 @@ TOKEN
 densegridencoder
 tets/256_tets.npz
 
+assets/
+launch.json
+settings.json
+
+aa*.png
```

Comment:
I think all these should be removed... e.g. launch.json is already in there, below (`.vscode/launch.json`)